mount/mount.go

```diff
@@ -22,3 +46,39 @@ func All(mounts []Mount, target string) error {
 	}
 	return nil
 }
+
+// TempMount mounts the provided mounts to a temp dir, and pass the temp dir to f.
```

Comment:
This needs to indicate that the mounts are valid for the duration of the closure.

Reply:
done